ODLA/platforms/magicmind/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# ==============================================================================
+# Copyright (C) 2020-2021 Alibaba Group Holding Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# ==============================================================================
+
+set(NEUWARE_HOME /usr/local/neuware/)
```

Comment:
can we change to something like:
```
if (NOT NEUWARE_HOME)
  set(NEUWARE_HOME "/usr/local/neuware/" CACHE PATH "Path for magicmind SDK")
endif()
```

so we can set the path like "cmake -DNEUWARE_HOME=....."